### ISSUE TITLE
`PackedSceneGLTF.import_gltf_scene()` not working in HTML5 platform

### ISSUE THREAD

**Issue by MrZak-dev** [godotengine/godot#60432]
### Godot version

3.5.beta4, 3.4.3.stable

### System information

HTML5, GLES3, GLES2

### Issue description

`PackedSceneGLTF.import_gltf_scene()` is not importing glb/gltf file at runtime when running HTML5 project export.

Note : it is working and importing glb/gltf file when running the project on desktop (windows)

### Steps to reproduce

- Create a project with 3D scene and attach a script
- import a glb/gltf file using 

```gdscript
func _ready() -> void:	
    var scene_importer = PackedSceneGLTF.new()
    var node = scene_importer.import_gltf_scene("res://debug_model.glb")
    add_child(node)
```
- run the project on desktop ( IT WORKS )
- export for HTML and run the project ( IT DOESN'T WORK ) with parsing this error to the browser console log 

```log
SCRIPT ERROR: Parse Error: The identifier "PackedSceneGLTF" isn't declared in the current scope.
    at: GDScript::load_byte_code (res://Spatial.gdc:11) - Parse Error: The identifier "PackedSceneGLTF" isn't declared in the current scope.
```

### Minimal reproduction project

[glb_runtime_importing.zip](https://github.com/godotengine/godot/files/8542041/glb_dynamic.zip)

**Comment by Calinou** [comment]
Run-time glTF loading and saving is not supported in exported projects in 3.x. It will only be supported in Godot 4.0.


If it works on desktop, it's because you're running the project from the editor, not the project exported for a desktop platform.

**Comment by akien-mga** [comment]
Fixed by #60437.
